contracts/modules/PermissionManager/GeneralPermissionManagerFactory.sol

```diff
@@ -26,7 +26,7 @@ contract GeneralPermissionManagerFactory is IModuleFactory {
         if(setupCost > 0)
             require(polyToken.transferFrom(msg.sender, owner, setupCost), "Failed transferFrom because of sufficent Allowance is not provided");
         address permissionManager = new GeneralPermissionManager(msg.sender, address(polyToken));
-        emit LogGenerateModuleFromFactory(address(permissionManager), getName(), address(this), msg.sender, now);
+        emit LogGenerateModuleFromFactory(address(permissionManager), getName(), address(this), msg.sender, setupCost, now);
         return address(permissionManager);
```

Comment:
or:
`address permissionManager = new ...`
should be:
PermissionManager permissionManager = new ...`

Reply:
Yes it could be only `permissionManager`. and for the getter functions it is used for CLI and stats scripts #194. I think can use directly the public functions but I think @VictorVicente was facing some problem regarding it.